cmd/treedb_text_hybrid_scale/main_test.go

```
@@ -848,3 +848,109 @@ func TestScaleCommandFlagValidation2731(t *testing.T) {
 		t.Fatalf("selected phases=%v want %v", got, wantSelected)
 	}
 }
+
+func TestManualProfileWrapperGuards4546(t *testing.T) {
+	run := func(env ...string) ([]byte, error) {
+		cmd := exec.Command("bash", "scripts/treedb_text_hybrid_scale_profile.sh")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)